### PR TITLE
Update PDFEmbed.hooks.php

### PR DESCRIPTION
**Pull request by lpc921**
Hi, I added a parameter for specifying a page to display the PDF document

### CODE REVIEW

**Comment by Alexia** [comment]
Do you mean on which page within the PDF to display in the preview and when it is initially opened?

**Comment by lpc921** [comment]
Should be the page when it is initially opened(preview?) within the browser. It only works with Firefox and Adobe's plug-in, but produces no side effects with other browsers and PDF object viewers. All it does is passing the parameter page=X to the PDF plugin. It's also easy to add support for bookmarks.

**Comment by lpc921** [comment]
I have fixed the support for Chrome and IE in the last commit. Now the #page=X is added to the "data" parameter.

File: extensions/PDFEmbed/PDFEmbed.hooks.php [hunk 1]
```diff
@@ -81,8 +82,8 @@ static public function generateTag($file, $args = [], Parser $parser, PPFrame $f
 	 * @param	integer	Height of the object.
 	 * @return	string	HTML object.
 	 */
-	static private function embed(File $file, $width, $height) {
-		return "<object width='{$width}' height='{$height}' data='".$file->getFullUrl()."' type='application/pdf'>".wfMessage('pdf_not_supported', $file->getFullUrl(), $file->getName())->plain()."</object>";
+	static private function embed(File $file, $width, $height, $page) {
+		return "<object width='{$width}' height='{$height}' data='".$file->getFullUrl()."#page={$page}"."' type='application/pdf'>".wfMessage('pdf_not_supported', $file->getFullUrl(), $file->getName())->plain()."<param name='page' value='$page' />"."</object>";
```
Comment (noahm) [inline extensions/PDFEmbed/PDFEmbed.hooks.php]: This line is starting to get unwieldy. I'd like to see it broken down into something more like:
```php
return Html::rawElement('object', [
  'width' => $width,
  'height' => $height,
  'data' => $file->getFullUrl().'#page='.$page,
  'type' => 'application/pdf'
],
  wfMessage('pdf_not_supported', $file->getFullUrl(), $file->getName())->escaped()
  ."<param name='page' value='$page' />"
);

Comment (lpc921) [inline extensions/PDFEmbed/PDFEmbed.hooks.php]: Thanks, that looks much better. I don't know much about php and HTML. I have applied you changes to the commit.

**Comment by noahm** [comment]
Ok, everything looks good to me. I'll give it a bit of testing on my end and then try to get this merged once Alexia shows back up.

**Comment by noahm** [comment]
OK, it doesn't do anything for the embedded PDF views in Safari on OS X but I also can't find any documentation on how (or if) specific pages are even supported there. Oh well.

Thanks for the contribution!

**Pull request merged**